index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Abdellah Portfolio</title>
+    <link rel="stylesheet" href="stylesheet.css">
+</head>
+<body>
+    <div class="firstSection">
+        <header class="myHeader">
+            <div id="my-logo">Welcome</div>
+            <div id="menu"><img src="img/menu.png" alt="menu icon" ></div>
+        </header>
+        <section class="main-content">
+            <h1 id="firstPara">Hey there.</h1><p id="supportingPara">I'm Abdou.</p>
+            <p id="secondPara">I'm a software developer</p>
+            <p id="supportingText"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hestiate to contact me.</p>
+            <div class="social-media-links">
+                <ul>
+                  <li><a href="https://twitter.com/belkaiduus" target="_blank" rel="noopener"><img rel="noreferrer" src="img/Twitter.png" alt="Twitter Logo" ></a></li>
+                  <li><a href="https://linkedin.com/in/belkaidus" target="_blank" rel="noopener"><img rel="noreferrer" src="img/Linkedin.png" alt="Linkedin Logo" ></a></li>
+                  <li><a href="https://github.com/mr-abdellah" target="_blank" rel="noopener"><img rel="noreferrer" src="img/GitHub.png" alt="GitHub Logo"></a></li>
+                </ul>
```

Comment:
- [ ] [OPTIONAL] Good job on implementing the `<ul>` 💯 as requested by the previous reviewer. Please, add the missing social icons and adjust the spacing in the "Hey, there. I am Abdou" to follow the figma design 🖼️ .

![image](https://user-images.githubusercontent.com/60979926/164333452-86a01691-9f69-4ebe-bb61-5ffb4097ee74.png)

Reply:
Hello thank you so much , i thought i just include just my social media accounts , and now i understand that i need to make a copy of the figma template , I'll try to fix it asap thanks for your kind feedback 😁👌🏻❤️

Reply:
No Problem, @mr-abdellah. Your attitude is exceptional. Keep up the Good Work 🚀.

Reply:
Thank you so much, I appreciate your help my brother @MrRamoun, the line spacing problem happened because I shrink my windows and I thought its the way to see your website with the mobile version then I remembered to use toggle device toolbar , again, thank you so much

Reply:
You are welcome @mr-abdellah. Please, anytime you have a technical question, DM me on my social accounts or slack (you can find the details in the [Review Message Body](https://github.com/mr-abdellah/my-portfolio/pull/2#pullrequestreview-947830390).